examples/plot_glm_retinal_ganglia.py

```diff
@@ -0,0 +1,421 @@
+# -*- coding: utf-8 -*-
+"""
+=================================================================
+GLM for Spike Trains Prediction in Primate Retinal Ganglion Cells
+=================================================================
+
+* Original tutorial adapted from Johnathan Pillow, Princeton University
+* Dataset provided by E.J. Chichilnisky, Stanford University
+* The dataset is granted by the original authors for the educational use in this tutorial
+* If you want to use it beyond educational purposes, please contact ``pillow@princeton.edu``
+
+First of all, we would like to thank Professor Johnathan Pillow and Professor E.J. Chichilnisky
+for sharing their tutorial dataset. The original MATLAB and Python tutorial can be found from
+https://github.com/pillowlab/GLMspiketraintutorial.
+
+These data were collected by Valerie Uzzell in the lab of
+E.J. Chichilnisky at the Salk Institute.  For full information see
+Uzzell et al. (J Neurophys 04), or Pillow et al. (J Neurosci 2005).
+**Again, the dataset in this tutorial is granted by the original authors
+for educational use only not for publication**. If you want to use it
+beyond the educational purposes,
+please contact ``pillow@princeton.edu``
+
+In this tutorial, we will demonstrate how to fit linear GLM and linear-nonlinear GLM 
+(a.k.a Poisson GLM) to predict the spike counts recorded from primate retinal ganglion cells.
+The dataset contains spike responses from 2 ON and 2 OFF parasol
+retinal ganglion cells (RGCs) in primate retina, stimulated with
+full-field `binary white noise`. Two experiment performed consisted of a
+long (20-minute) binary stochastic (non-repeating) stimulus
+which can be used for computing the spike-triggered average
+(or characterizing some other model of the response).
+"""
+
+# Authors: Jonathan Pillow <pillow@princeton.edu>
+# Edited by: Titipat Achakulvisut <my.titipat@gmail.com>, Konrad Kording <koerding@gmail.com>
```

Comment:
just list yourself as authors, no need to say "edited by". We are in open source world.